inc/class-core-sitemaps-renderer.php

```diff
@@ -48,6 +72,13 @@ public function render_index( $sitemaps ) {
 		// All output is escaped within the addChild method calls.
 		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		echo $sitemap_index->asXML();
+
+		/**
```

Comment:
I don't think this is the right location for this filter. Did you mean for it to be in `get_sitemap_stylesheet_url()` instead?